core/pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <groupId>com.mindscapehq</groupId>
         <artifactId>raygun4java</artifactId>
-        <version>4.0.0-rc2-SNAPSHOT</version>
+        <version>4.0.0</version>
```

Comment:
Irrelevant to the PR, do you want do first an rc2 release just to confirm the release process and everything?